src/guides/v2.4/graphql/mutations/copy-items-between-requisition-lists.md

```diff
@@ -0,0 +1,96 @@
+---
+group: graphql
+title: copyItemsBetweenRequisitionLists mutation
+b2b_only: true
+contributor_name: EY
+---
+The `copyItemsBetweenRequisitionLists` mutation copies items from one requisition list to another.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+{:.bs-callout-info}
+Use the [storeConfig query]({{page.baseurl}}/graphql/queries/store-config.html) with the `btob_website_configuration_requisition_list_active` attribute to determine whether requisition lists are supported.
+
+## Syntax
+
+```graphql
+mutation {
+  copyItemsBetweenRequisitionLists(
+    sourceRequisitionListUid: ID!,
+    destinationRequisitionListUid: ID,
+    requisitionListItem: CopyItemsBetweenRequisitionListsInput
+  ) {
+    CopyItemsFromRequisitionListsOutput
+  }
+}
+```
+
+## Example usage
+
+The following example copies items from one requisition list to another.
+
+**Request:**
+
+``` graphql
+mutation {
```

Comment:
This code sample won't work. `requisitionListItem` is supposed to be within a `requisitionListItemUids` object. In addition, it looks like the requisition list Uids are b64 encode values.